src/RazorSdk/Sdk/Sdk.targets

```diff
@@ -14,7 +14,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.targets" Condition="'$(_RazorSdkImportsMicrosoftNetSdk)' == 'true'" />
 
   <PropertyGroup Condition="'$(RazorSdkCurrentVersionTargets)' == ''">
-    <RazorSdkCurrentVersionTargets Condition="'$(IsCrossTargetingBuild)' != 'true'">$(MSBuildThisFileDirectory)..\build\netstandard2.0\Sdk.Razor.CurrentVersion.targets</RazorSdkCurrentVersionTargets>
+    <RazorSdkCurrentVersionTargets>$(MSBuildThisFileDirectory)..\build\netstandard2.0\Sdk.Razor.CurrentVersion.targets</RazorSdkCurrentVersionTargets>
```

Comment:
hmm?

Reply:
When we introduced this PR, we removed the cross-targeting references so I don't think we need to this anymore. We'll always import these targets, correct?